packages/console/src/components/CopyToClipboard/index.tsx

```diff
@@ -41,7 +56,14 @@ const CopyToClipboard = ({ value, className, variant = 'contained' }: Props) =>
       }}
     >
       <div className={styles.row}>
-        {variant === 'icon' ? null : value}
+        {variant === 'icon' ? null : <div className={styles.content}>{valueToShow}</div>}
+        {isContentHidden && (
+          <div className={styles.eye}>
+            <IconButton onClick={toggleHiddenContent}>
+              <Eye />
+            </IconButton>
+          </div>
+        )}
```

Comment:
Should have a "closed eye" icon when hiding the secret

Reply:
updated, PTAL

Reply:
Should update the screenshot in PR description as well. Cheers

Reply:
ha, I add a comment to keep the modify history.